packages/launcher/src/main/rpc/app.js

```diff
@@ -53,6 +51,15 @@ const getStorageManifestHash = async (
   }
 }
 
+const raiseErrorIfKeyInvalid = (key: string): void | Error => {
```

Comment:
If the error is thrown, the returned type here should only be `void`, as throwing an error is an exception.
This return type would work if `return new Error()` was used rather than `throw new Error()`.